### ISSUE TITLE
A better error message in the case where a log group already has a subscription

### ISSUE THREAD

**Issue by justinmchase** [tsub/serverless-plugin-subscription-filter#3]
I recently hit an issue where my deployment was failing with the error:

> An error occurred while provisioning your stack
>     - Resource limit exceeded..

Being relatively new to serverless and AWS I didn't know what this message meant. It turns out after [getting some help](https://github.com/serverless/serverless/issues/2605#issuecomment-283575064) that I had already manually created a subscription filter for one of my lambdas and since I can only have one this error was occurring. It would be nice have a slightly more robust error message in this case. Something like below possibly:

```
    - Resource limit exceeded..

      You've hit a AWS resource limit:
      http://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/cloudwatch_limits_cwl.html

      Subscription filters: 1/log group. This limit cannot be changed.
```

**Comment by tsub** [comment]
I'm sorry I left it for a long time.
I think that the point is correct, so I will respond in the near future.

**Comment by elyobo** [comment]
Bit me too, thanks for the note @justinmchase

**Comment by tsub** [comment]
@justinmchase 

It implemented in PR https://github.com/tsub/serverless-plugin-subscription-filter/pull/7.
And it was released in v1.0.4 earlier.
